File: TaskManager/TaskManager_210042112/TaskManager_112/TaskManager_112/CreateIssueCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager_112
{
    public class CreateIssueCommand : ICommand
    {
        private IssueRepository repository;
        private Issue issue;
        public CreateIssueCommand(IssueRepository repo, Issue issue)
        {
            repository = repo;
            this.issue = issue;
        }
        public void Execute()
        {
            repository.CreateIssue(issue);
        }
        public void Undo()
        {
            Console.WriteLine($"Undid the last created issue: {issue.ID}");
        }
    }
}

File: IssueTracker_210042122/src/Class/Commands/CreateIssueCommand.java
package Class.Commands;


import Class.*;

public class CreateIssueCommand implements Command {
    private IssueRepository issueRepository;
    private Issue issue;


    public CreateIssueCommand(IssueRepository issueRepository, Issue issue) {
        this.issueRepository=issueRepository;
        this.issue = issue;
    }

    @Override
    public void Execute() {
        issueRepository.CreateIssue( issue);

    }



    @Override
    public void Undo() {
        issueRepository.RemoveIssue(issue);
    }
}
